src/material-experimental/mdc-progress-spinner/progress-spinner.scss

```diff
@@ -8,6 +8,11 @@
   overflow: hidden;
 }
 
-:not(._mat-animation-noopable) {
+.mat-mdc-progress-spinner:not(._mat-animation-noopable) {
   @include mdc-circular-progress-core-styles($query: animation);
 }
+
+// Render the indeterminate spinner as a complete circle when animations are off
+._mat-animation-noopable .mdc-circular-progress__indeterminate-container circle {
```

Comment:
I think you can target `._mat-animation-noopable .mdc-circular-progress__determinate-circle` directly. The specificity shouldn't matter since you're using `!important`.

Reply:
I was hoping so, but they apply the styles directly to the element dynamically so there's no way to override it via CSS unless we use `!important`

Reply:
I meant that changing the selector to the one I mentioned would reduce the specificity, but it won't matter if we keep the `!important`.